src/pages/_app.js

```diff
@@ -7,19 +9,28 @@ import theme from "../assets/theme";
 
 import { Loader } from "../components/loader";
 // import NavBar from "../components/nav-bar";
-import NavBar from "../components/Navbar/Navbar";
-import Footer from "../components/Footer/Footer";
+// import NavBar from "../components/Navbar/Navbar";
+
+// TODO: Set up MUI for server side rendering: https://github.com/mui/material-ui/tree/HEAD/examples/material-next
+const NavBar = dynamic(() => import('../components/Navbar/Navbar'), {
+  ssr: false,
+})
+
+const Footer = dynamic(() => import('../components/Footer/Footer'), {
+  ssr: false,
+});
 import Head from "next/head";
 
-import * as ga from "../lib/ga";
-import { useRouter } from "next/router";
-import { useEffect } from "react";
+// import * as ga from "../lib/ga";
```

Comment:
@rstackhouse - I think this removes Google Analytics from the website - I don't want to merge into main without a solve here